### PR TITLE
Introducing Direct Method Sender to Connectivity Test

### PR DESCRIPTION
**Pull request by yophilav**
- Introduce the connectivity test parameters to Direct Method Sender Deployment

### CODE REVIEW

File: e2e_deployment_files/connectivity_deployment.template.json [hunk 1]
```diff
@@ -107,6 +107,107 @@
...
+            }
+          },
+          "DirectMethodSender2": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "ClientTransportType" : {
+                "value": "Amqp_Tcp_Only"
```
Comment (philipktlin) [inline e2e_deployment_files/connectivity_deployment.template.json]: mqtt

File: e2e_deployment_files/connectivity_deployment.template.json [hunk 2]
```diff
@@ -107,6 +107,107 @@
...
+            }
+          },
+          "DirectMethodReceiver2": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "ClientTransportType": {
+                "value": "Amqp_Tcp_Only"
```
Comment (philipktlin) [inline e2e_deployment_files/connectivity_deployment.template.json]: mqtt

File: e2e_deployment_files/connectivity_deployment.template.json [hunk 3]
```diff
@@ -240,7 +341,8 @@
           "loadGen1ToRelayer1": "FROM /messages/modules/loadGen1/outputs/output1 INTO BrokeredEndpoint(\"/modules/relayer1/inputs/input1\")",
           "relayer1ToCloud": "FROM /messages/modules/relayer1/outputs/output1 INTO $upstream",
           "loadGen2ToRelayer2": "FROM /messages/modules/loadGen2/outputs/output2 INTO BrokeredEndpoint(\"/modules/relayer2/inputs/input2\")",
-          "relayer2ToCloud": "FROM /messages/modules/relayer2/outputs/output2 INTO $upstream"
+          "relayer2ToCloud": "FROM /messages/modules/relayer2/outputs/output2 INTO $upstream",
+          "directMethodSender1ToUpstream": "FROM /messages/modules/DirectMethodSender/outputs/output1 INTO $upstream"
```
Comment (philipktlin) [inline e2e_deployment_files/connectivity_deployment.template.json]: I don't think we need this, direct method cloud sender previously report success to iot hub by sending event message.  However now both long haul and connectivity tests, reporting result is no longer using even message.  Please remove it here and have another PR to remove from long haul deployment file.

**Comment by philipktlin** [comment]
Either set status of DM sender modules in deployment to "stopped" or I think it is better to add other changes to make DM sender to use the new parametres.

File: e2e_deployment_files/connectivity_deployment.template.json (hunk 1 above)
Comment (and-rewsmith) [reply e2e_deployment_files/connectivity_deployment.template.json]: Do direct methods support amqp?

Comment (philipktlin) [reply e2e_deployment_files/connectivity_deployment.template.json]: yes. why not?

**Pull request closed without merging**